scripts/web/web-repack.sh

```diff
@@ -59,23 +65,31 @@ PROJECTNAME=$(cd "${SOURCE}" && ls *_*.js | head -1 | sed 's,_.*.js$,,')
 echo $PROJECTNAME
 
 TARGET="${SOURCE}.repack"
-
-echo "$TARGET"
+echo "$TARGET [${VARIANT}]"
 
 if [ -d "$TARGET" ]; then
     rm -rf "$TARGET"
 fi
 
+SUFFIX=""
+[ "$VARIANT" != "debug" ] && SUFFIX="_${VARIANT}"
+
 cp -r "${SOURCE}/" "${TARGET}/"
 
 if [ -d "${DYNAMO_HOME}/bin/js-web" ]; then
-    cp -v "${DYNAMO_HOME}/bin/js-web/dmengine.js" "${TARGET}/${PROJECTNAME}_asmjs.js"
+    cp -v "${DYNAMO_HOME}/bin/js-web/dmengine${SUFFIX}.js" "${TARGET}/${PROJECTNAME}_asmjs.js"
+    if [ -e "${DYNAMO_HOME}/bin/js-web/dmengine${SUFFIX}.wasm.map" ]; then
+        cp -v "${DYNAMO_HOME}/bin/js-web/dmengine${SUFFIX}.js.symbols" "${TARGET}/${PROJECTNAME}_asmjs.symbols"
+    fi
 fi
 if [ -d "${DYNAMO_HOME}/bin/wasm-web" ]; then
-    cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine.js" "${TARGET}/${PROJECTNAME}_wasm.js"
-    cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine.wasm" "${TARGET}/${PROJECTNAME}.wasm"
-    if [ -e "${DYNAMO_HOME}/bin/wasm-web/dmengine.wasm.map" ]; then
-        cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine.wasm.map" "${TARGET}/dmengine.wasm.map"
+    cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine${SUFFIX}.js" "${TARGET}/${PROJECTNAME}_wasm.js"
+    cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine${SUFFIX}.wasm" "${TARGET}/${PROJECTNAME}.wasm"
+    if [ -e "${DYNAMO_HOME}/bin/wasm-web/dmengine${SUFFIX}.wasm.debug.wasm" ]; then
+        cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine${SUFFIX}.wasm.debug.wasm" "${TARGET}/${PROJECTNAME}.wasm.debug.wasm"
+    fi
+    if [ -e "${DYNAMO_HOME}/bin/wasm-web/dmengine${SUFFIX}.wasm.map" ]; then
+        cp -v "${DYNAMO_HOME}/bin/wasm-web/dmengine${SUFFIX}.wasm.map" "${TARGET}/${PROJECTNAME}.wasm.map"
```

Comment:
❤️ 

Soon I hope we can get this stuff available for the average user as well, in the local builds from bob/editor